Create the mobile website section to list the portfolio projects

d4nQw3rty opened the pull request:
*Create the main container using CSS Grid to contain all the cards
*Use Flexbox to lay out an individual card.
*Implement the button interactions (enable, hover, etc).
*Build About myself section.

d4nQw3rty committed:
Create Counter.png

* add counter image

d4nQw3rty committed:
Create Snapshoot-Portfolio.png

* create snapshoot-portfolio

d4nQw3rty committed:
Update Thumbs.db

*add image to img

d4nQw3rty committed:
Update index.html

*add new section for project.
*add 4 espace to hold project information
*add information to first proyect
*

d4nQw3rty committed:
update css

*add main-container, and items.
*add grid and project title css
*add list class for ul li
*add text  and  button class

d4nQw3rty committed:
update index.html, style.css

*add button and hover for button
*build work section as grid
*

d4nQw3rty committed:
add Portfolio 2 snapshot images

*add second portfolio snapshot image.

d4nQw3rty committed:
update img

*add portfolio 3 and 4 img.

d4nQw3rty committed:
update img file

*add down arrow image, right arrow, image

d4nQw3rty committed:
Update index.html

*Update index.html

d4nQw3rty committed:
Update style.css

*update style.css

d4nQw3rty committed:
Update index.html

*fix lets connect headline

d4nQw3rty committed:
update index.html, style.css

*Fixed linters alert.

d4nQw3rty committed:
add languages for language list

* add img for lavascript, html and css  language in for languge skill
*update index.html for language list
*update style.css for language list

d4nQw3rty committed:
Update index.html, style.css

*Fix linter errors.

vechicin commented on index.html:
```diff
@@ -7,39 +7,185 @@
+                          <li class="tag-css"><P>css</P></li>
+                          <li class="tag-js"><p>javaScript</p></li>
+                      </ul>
+                       <button type="button" class="button-enable button-enable-text" >See Project</button>
```
Your buttons don't have the required interaction stylings. Please add them to all.

<img width="832" alt="Screen Shot 2022-07-07 at 12 06 07" src="https://user-images.githubusercontent.com/82426682/177830497-25da6de8-92e0-4cb9-9e3d-ae982020fd4f.png">

vechicin commented on index.html:
```diff
@@ -7,39 +7,185 @@
+                  <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+              </ul>                                  
+         </section>    
+         <section id="main-container">          
```
This section doesn't have the margins defined in the Figma template. Please fix them.

<img width="227" alt="Screen Shot 2022-07-07 at 12 07 13" src="https://user-images.githubusercontent.com/82426682/177830662-6c5a0a72-f7b8-4fe7-a213-1ed3311e848d.png">

<img width="488" alt="Screen Shot 2022-07-07 at 12 07 31" src="https://user-images.githubusercontent.com/82426682/177830705-4211468a-8cc1-4d24-8fb5-44a84cc59484.png">

vechicin commented on index.html:
```diff
@@ -7,39 +7,185 @@
+                <p class="Primary-Content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>          
+                <div class="Social-Media">
+                    <p class="Lets-Connect">LET’S CONNECT</p>            
```
This content should be wrapped in an `<hx>`.

<img width="439" alt="Screen Shot 2022-07-07 at 12 23 15" src="https://user-images.githubusercontent.com/82426682/177833308-36fa2240-88d6-40e0-870c-1811219b4de1.png">

vechicin commented on index.html:
```diff
@@ -7,39 +7,185 @@
+                     <li><a href="https://medium.com/" target="_blank" rel="noopener noreferer" ><img src="img/Vector-M.svg" alt="Medium link"></a></li>
+                     <li><a href="https://github.com/d4nQw3rty" target="_blank" rel="noopener noreferrer"><img src="img/Vector-github.svg" alt="Github link"></a></li>
+                     <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+                 </ul>
```
The order of the icons doesn't match that of the template design. Please take a look at the Figma template and fix it.

vechicin commented on index.html:
```diff
@@ -7,39 +7,185 @@
+                     <li><a href="https://www.angellist.com/" target="_blank" rel="noopener noreferrer"><img src="img/Vector-hand.svg" alt="angellist link"></a></li>
+                 </ul>
+                 <div id="about-myself-button-holder">
+                      <button type="button" class="button-enable button-enable-text">Get my resume</button>
```
Your button doesn't have interaction styling. Please take a look at the previous comment and fix it.

vechicin commented on style.css:
```diff
@@ -190,3 +193,620 @@
+
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
```
None of your `font-weight` rules are working. That is most likely caused by not importing the correct font weight from Google Fonts. Please refer to [this video](https://www.youtube.com/watch?v=mRoBBOTYNz4&t=184s) to fix this issue.

<img width="560" alt="Screen Shot 2022-07-07 at 13 03 24" src="https://user-images.githubusercontent.com/82426682/177839886-46fd92fa-3d02-45e2-8859-e546b760a981.png">

vechicin reviewed (changes requested):
# Hi @d4nQw3rty ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

d4nQw3rty committed:
Update style.css

**Fix main container section margin
**update background color to match with figma template.

d4nQw3rty committed:
Update style.css

**fix linter alert!

d4nQw3rty committed:
Merge pull request #7 from d4nQw3rty/fix-button-styling

Update index.html and style.css

d4nQw3rty committed:
Update index.html and style.css

**Fixed button style enable, hover and focus

d4nQw3rty committed:
Update style.css

*fix lint error

d4nQw3rty committed:
Update style.css

*fix  main-conteiner  margin

d4nQw3rty committed:
Update index.html and style.css

**Change <p class="Lets-Connect">LET’S CONNECT</p>
                                               to
               <h2 class="Lets-Connect">LET’S CONNECT</h2>

**Fix linters error

d4nQw3rty committed:
Update index.html and style.css

*Fix About myself section icon

d4nQw3rty committed:
Fix main-container margin problem

*update index.html

*update style.css

victorokoroji reviewed (approved):
Hi @d4nQw3rty , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

d4nQw3rty committed:
update index.html and style.css

**Fix about me icons.
